src/Controls/tests/TestCases.Shared.Tests/Tests/Issues/Bugzilla/Bugzilla45926.cs

```diff
@@ -6,30 +6,44 @@ namespace Microsoft.Maui.TestCases.Tests.Issues;
 
 public class Bugzilla45926 : _IssuesUITest
 {
+#if ANDROID
+	const string BackButtonIdentifier1 = "";
+	const string BackButtonIdentifier2 = "";
+#else
+	const string BackButtonIdentifier1 = "Back";
+	const string BackButtonIdentifier2 = "Test";
+#endif
+
 	public Bugzilla45926(TestDevice testDevice) : base(testDevice)
 	{
 	}
 
-	public override string Issue => "Effect not attaching to ScrollView";
-
-	// TODO Xamarin.UITest Migration
-	// [Test]
-	// [FailsOnAndroidWhenRunningOnXamarinUITest]
-	// public void Issue45926Test()
-	// {
-	// 	App.WaitForElement(q => q.Marked("New Page"));
-
-	// 	App.Tap(q => q.Marked("New Page"));
-	// 	App.WaitForElement(q => q.Marked("Second Page #1"));
-	// 	App.Back();
-	// 	App.WaitForElement(q => q.Marked("Intermediate Page"));
-	// 	App.Back();
-	// 	App.Tap(q => q.Marked("Do GC"));
-	// 	App.Tap(q => q.Marked("Do GC"));
-	// 	App.Tap(q => q.Marked("Send Message"));
-	// 	App.Tap(q => q.Marked("Do GC"));
-
-	// 	App.WaitForElement(q => q.Marked("Instances: 0"));
-	// 	App.WaitForElement(q => q.Marked("Messages: 0"));
-	// }
+	public override string Issue => "MessagingCenter prevents subscriber from being collected";
+
+	[Test]
+	[Category(UITestCategories.Page)]
+	public void Issue45926Test()
```

Comment:
This test is failing on Android:
<img width="1143" alt="image" src="https://github.com/user-attachments/assets/ed0cc703-5659-434e-bd19-5afbc7919c11">

Reply:
Resolved CI failure and changes are committed.
<img width="392" alt="image" src="https://github.com/user-attachments/assets/c36c7341-3d7f-4efe-bdfa-0f1545fb8c2f">